src/ntttcp-for-linux-log.xml

```diff
@@ -0,0 +1,59 @@
+<ntttcpr computername="centos8.3" version="5.33-linux">
```

Comment:
The same, please exclude this file from your commit.

Reply:
Has been removed.